projects/pcre2/build.sh

```diff
@@ -21,6 +21,8 @@
 make -j$(nproc) clean
 make -j$(nproc) all
 
+cp $SRC/pcre2_fuzzer.dict $SRC/pcre2_fuzzer.options $OUT/
```

Comment:
I don't see .options file in the pcre2 repo: https://github.com/svn2github/pcre2